working_time/working_time/report/expected_and_actual_working_time/expected_and_actual_working_time.js

```diff
@@ -30,4 +32,32 @@ frappe.query_reports["Expected and Actual Working Time"] = {
 			reqd: 1,
 		},
 	],
+	// fetch filter values for employee and daily_working_hours using db call and python script
+	onload: function(report) {
+		frappe.call({
+			method: "working_time.working_time.report.expected_and_actual_working_time.get_filter_values.get_employee_working_hours",
+			args: {
+				"user": frappe.session.user
+			},
+			callback: (r) => {
+				if (r.message) {
+					var filter = report.get_filter("daily_working_hours");
+					filter.set_value(r.message);
+				}
+			}
+		});
```

Comment:
```suggestion
		frappe.call({
			method: "working_time.working_time.report.expected_and_actual_working_time.get_filter_values.get_employee_working_hours",
			args: {
				"employee": report.get_filter_value("employee")
			},
			callback: (r) => {
				if (r.message) {
					var filter = report.get_filter("daily_working_hours");
					filter.set_value(r.message);
				}
			}
		});
```

- Pass the selected employee as filter
- This should get called whenever the employee filter changes